index.html

```diff
@@ -1,117 +1,120 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>판다마켓</title>
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
     <!-- 헤더 -->
     <header class="header">
-        <div class="header-content">
-            <a href="/">
-                <img class="logoImg" src="/images/logo.png" alt="logo">
-            </a>
-           
-            <a href="/pages/login.html" class="header-button button">로그인</a>
-        </div>
+      <div class="header-content">
+        <a href="/">
+          <img class="logoImg" src="/images/logo.png" alt="판다마켓" />
+        </a>
+
+        <a href="/pages/login.html" class="header-button button">로그인</a>
+      </div>
     </header>
- <!-- 상단랜딩 -->
+    <!-- 상단랜딩 -->
     <div class="topLanding">
-        <div class="top-content">
-            <div class="top-content-text">
-                <h2>일상의 모든 물건을<br>거래해 보세요</h2>
-                <a href="/pages/items.html" class="items-button button">구경하러 가기</a>
-
-            </div>
-           
-            <img  class="topLandingImg" src="/images/topImg.png" alt="topImg">
-
+      <div class="top-content">
+        <div class="top-content-text">
+          <h2>일상의 모든 물건을<br />거래해 보세요</h2>
+          <a href="/pages/items.html" class="items-button button"
+            >구경하러 가기</a
+          >
```

Comment:
💊 제안
현재 코드에서는 하나의 태그가 분리되어 있어 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~


```suggestion
          <a href="/pages/items.html" class="items-button button">
            구경하러 가기
          </a>
```